pkg/webui/lib/errors/utils.js

```diff
@@ -501,12 +501,22 @@ export const toMessageProps = (error, each = false) => {
       if (hasCauses(errorDetails)) {
         // Use the root cause if any.
         const rootCause = getBackendErrorRootCause(errorDetails)
+
+        // If the attributes are missing values, use the generated default values based on the message.
+        const messageValues = rootCause.message_format.match(/[^{}]+(?=})/g) || []
+        const hasMissingValues =
+          rootCause.attributes && Object.keys(rootCause.attributes).length < messageValues.length
+        const missingValues = messageValues.filter(val => !(val in rootCause.attributes))
+        const defaultErrorValues = hasMissingValues
+          ? missingValues.reduce((obj, val) => ((obj[val] = `{${val}}`), obj), {})
+          : {}
```

Comment:
```suggestion
        const messageValues = rootCause?.message_format?.match(/[^{}]+(?=})/g) || []
        const fallbackValues = messageValues.reduce((obj, val) => ((obj[val] = `{${val}}`), obj), { ...rootCause.attributes })
        const values = { ...fallbackValues, ...rootCause.attributes }
```
Let's just always spread the existing values over the missing values.